test/SparqlParserTest.cpp

```diff
@@ -556,8 +556,10 @@ TEST(ParserTest, testParse) {
       ASSERT_EQ(vvars, sc.getSelectedVariablesAsStrings());
 
       // -- SubQuery
-      auto parsed_sub_query = get<GraphPatternOperation::Subquery>(
+      auto subQueryGroup = get<GraphPatternOperation::GroupGraphPattern>(
           pq._rootGraphPattern._children[1].variant_);
+      auto parsed_sub_query = get<GraphPatternOperation::Subquery>(
+          subQueryGroup._child._children[0].variant_);
```

Comment:
Can you quickly confirm that this is only because now
`{ SELECT ... }` is parsed as a Subquery INSIDE an explicit GroupGraphPattern {}, which was not there in the previous structure.

Reply:
Yes that is correct. This is how its specified in the grammar. The old parser optimized the extra `GroupGraphPattern` away. I'll look into how easy it is to also do this in the new parser.